aws/util.sh

```diff
@@ -5,7 +5,40 @@
 # All providers are expected to support the following functions:
 #	util::get_instance_info
 #	util::copy_file
-#
+
+# Helpers #
+# These functions implement lower leverl operations required for deploying
+# and destroying Amazon Web Service EC2 instances.  These functions should be
+# considered private, scoped to the provider.
+
+function __run_instance() {
```

Comment:
My proposed func name convention is "util::<func-name>". So this func would be named "util::run_instance", if you like that convention. My idea was to include the helper file name as the func prefix, eg "util::" and "init::"

Reply:
I do like that convention.  Do you think there's value in denoting "private" functions (provider specific) from public high level ops.   That was my intent with the underbar prefix.

Reply:
I do like that convention.  Do you think there's value in denoting "private" functions (provider specific) from public high level ops.   That was my intent with the underbar prefix.

Reply:
We're not providing a package per se so all funcs in the helper files are callable. For now I do not think it's worth breaking the naming convention to designate lower level funcs...